src/components/Proposal/ViewProposalContainer.tsx

```diff
@@ -40,23 +40,31 @@ type IProps = IStateProps & IDispatchProps;
 
 class ViewProposalContainer extends React.Component<IProps, null> {
 
-  public componentDidMount() {
-    this.props.getProposal(this.props.proposal.daoAvatarAddress, this.props.proposal.proposalId);
-  }
+  public componentDidMount() {}
 
   public render() {
     const { proposal } = this.props;
 
+    const disqusShortname = 'daostack-alchemy';
+    const disqusConfig = {
```

Comment:
seems like this could be extracted to a constant somewhere (used more than once).

Reply:
truth